server/public/styles.css

```diff
@@ -0,0 +1,107 @@
+* {
+  box-sizing: border-box;
+  font-family: 'Roboto', sans-serif;
+}
+
+header, footer, main {
+  max-width: 1024px;
+  margin: 0 auto;
+}
+
+html,
+body {
+  height: 100%;
+}
+
+body {
+  display: flex;
+  flex-direction: column;
+}
+
+main {
+  flex: 1 0 auto;
+  position: relative;
+  min-height: calc(100vh - 146px);
+}
+
+footer {
+  flex-shrink: 0;
+}
+
+main.home::after{
+  content: "";
+  background-image: url(background.jpg);
+  background-size: cover;
+  position: absolute;
+  top: 0px;
+  right: 0px;
+  bottom: 0px;
+  left: 0px;
+  opacity: 0.50;
+  z-index: -1;
+}
+
+h3 {
+  font-size: 2rem;
+  letter-spacing: 1.5px;
+  font-weight: 400;
+}
+
+h4 {
+  font-size: 1.5rem;
+}
+
+a.a-footer {
+  font-size: 1.5rem;
+  text-decoration: none;
+}
+
+button:hover, a:hover {
+  cursor: pointer;
+}
+a:visited, a:hover, a:link, a:active{
+  text-decoration: none;
+  color: inherit;
+}
+
+i.size1 {
+  font-size: 1.6rem;
+  margin: .5rem;
```

Comment:
Bootstrap has margin helpers, you should stick with those if they don't break the layout.